site/docs/reference/Connectors/materialization-connectors/Rockset.md

```diff
@@ -42,4 +42,123 @@ materializations:
+`}/>
+
+To set this up, use the following procedure as a guide, substituting `example/flow/collection` for your collection:
+
+1. Follow the [instructions here](https://rockset.com/docs/amazon-s3/#create-an-s3-integration) to create the integration, but _do not create the Rockset collection yet_.
+2. Create and activate a materialization of `example/flow/collection` into a unique prefix within an S3 bucket of your choosing.
+  ```yaml
+  materializations:
+    example/toRockset:
+      endpoint:
+        connector:
+          image: ghcr.io/estuary/materialize-s3-parquet:dev
+          config:
+            bucket: example-s3-bucket
+            region: us-east-1
+            awsAccessKeyId: <your key>
+            awsSecretAccessKey: <your secret>
+            uploadIntervalInSeconds: 300
+      bindings:
+        - resource:
+            pathPrefix: example/s3-prefix/
+          source: example/flow/collection
+  ```
+3. Once the  S3 materialization is caught up with your historical data, you'll switch to the Rockset write API for your future data. To make the switch, first disable the S3 materialization by setting shards to disabled in the definition, and re-deploy the catalog. This is necessary to ensure correct ordering of documents written to Rockset.
+  ```yaml
+  materializations:
+    example/toRockset:
+      shards:
+        disable: true
+      # ...the remainder of the materialization yaml remains the same as above
+  ```
+4. Update the materialization to use the `materialize-rockset` connector, and re-enable the shards. Here you'll provide the name of the Rockset S3 integration you created above, as well as the bucket and prefix that you previously materialized into. **It's critical that the name of the materialization remains the same as it was for materializing into S3.**
+  ```yaml
+  materializations:
+    example/toRockset:
+      endpoint:
+        connector:
+          image: ghcr.io/estuary/materialize-rockset:dev
+          config:
+            api_key: <your rockset API key here>
+            max_concurrent_requests: 5
+      bindings:
+        - resource:
+            workspace: <your rockset workspace name>
+            collection: <your rockset collection name>
+            initializeFromS3:
+              integration: <rockset integration name>
+              bucket: example-s3-bucket
+              region: us-east-1
+              prefix: example/s3-prefix/
+          source: example/flow/collection
+  ```
+5. When you activate the new materialization, the connector will create the Rockset collection using the given integration, and wait for it to ingest all of the data from S3 before it continues. During this time, the Flow shards will remain in `STANDBY` status, so `flowctl deploy` is expected to block until the bulk ingestion completes. Once this completes, the materialize-rockset connector will automatically switch over to using the Rockset write API.
+
+## Potential improvements
```

Comment:
The README in the connectors repo was written more with a developer audience in mind. I think this section is better left out of this, more user-facing, doc. This could be a good place to document the "escape hatch" described above, though. Maybe something like:

If you want complete control over how the Rockset Collection is created, then you can always create it yourself in the Rockset console, and Flow's materialize-rockset collection will happily use it as long as the name matches.

Reply:
As above, I just took the whole thing out. In general, I've simplified this pretty heavily so as not to provide more info than necessary to the end user. Maybe give it one more read when you can, and then I'll merge?